packages/components/src/composite/stories/index.story.tsx

```diff
@@ -353,3 +354,50 @@ const Fill = ( { children } ) => {
 		},
 	},
 };
+
+/**
+ * Combining the `Tooltip` and `Composite` component has a few caveats.
+ * For example, the following code won't work as expected:
+ *
+ * ```jsx
+ * <Composite.Item
+ *   render={
+ *   <Tooltip text="Tooltip">
+ *     <button>Button</button>
+ *   </Tooltip>
```

Comment:
Should we make this a more complete example? It's lacking the closing markup (`}` and `</Composite.Item>`.